config/jail.conf

```diff
@@ -767,3 +767,16 @@ port = 2222
 enabled  = false
 logpath  = /var/lib/portsentry/portsentry.history
 maxretry = 1
+
+[pass2allow-ftp]
+# this pass2allow example allows FTP traffic after successful HTTP authentication
+port         = ftp,ftp-data,ftps,ftps-data
+# knocking URL needs to be customized in apache-pass.local
```

Comment:
"needs to be" or "can be". In direct reference to your comment in jail.conf...this very line which I am commenting on in github...which says "knocking URL needs to be customized"...although a default is already provided...so an override CAN be provided IF the user CHOOSES. So does it "need" to be customized? Or "can" it be customized?

I have no idea why you mention @yarikoptic here.